data/json/items/ammo/223.json

```diff
@@ -6,6 +6,7 @@
     "description": ".223 Remington ammunition with 55gr FMJ bullets.  The .223 round has been very popular with civilian shooters for almost a century, finding use in a wide variety of weapons.  It generates lower pressure than 5.56 NATO leading to slightly decreased accuracy and recoil.",
     "weight": "12 g",
     "volume": "194 ml",
+    "longest_side": "5.75 cm",
```

Comment:
```suggestion
    "longest_side": "58 mm",
```

Reply:
`mm` [is the smallest unit](https://discord.com/channels/598523535169945603/598529174302490644/827820962740568075), maybe it would be great to include an improvement to the https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/JSON_INFO.md#units

Reply:
https://github.com/CleverRaven/Cataclysm-DDA/blob/master/src/units.h#L886